github/ci/services/grafana/deploy/base/grafana.yaml

```diff
@@ -54,7 +54,7 @@ kind: GrafanaDatasource
 metadata:
   annotations: {}
   name: prometheus-grafanadatasource
-  namespace: monitoring
+  namespace: default
```

Comment:
why is this namespace change needed?

Reply:
IIRC this is present so it can be overridden in the kustomize, but would need to check.

Reply:
Instead of having scattered namespace declarations that seem to
conflict I moved the namespace declaration into the base kustomize
and removed it in the grafana.yaml.